tree/dataframe/test/dataframe_cache.py

```diff
@@ -1,36 +1,46 @@
 import unittest
 import ROOT
+import os
 
 RDataFrame = ROOT.ROOT.RDataFrame
 
 class Cache(unittest.TestCase):
     @classmethod
     def setUp(cls):
-      code = ''' {
-      const char* treeName = "t";
-      const char* fileName = "fileName.root";
-      TFile f(fileName, "RECREATE");
-      TTree t(treeName, treeName);
-      float arr[4];
-      t.Branch("arr", arr, "arr[4]/F");
-      for (auto i : ROOT::TSeqU(4)) {
-         for (auto j : ROOT::TSeqU(4)) {
-            arr[j] = i + j;
-         }
-         t.Fill();
-      }
-      t.Write();
-      }'''
-      ROOT.gInterpreter.Calc(code)
+        # Necessary because old and new Cppyy use a different syntax for
+        # overloaded templated methods (round brackets the former,
+        # square brackets the latter)
+        cls.exp_pyroot = os.environ.get('EXP_PYROOT') == 'ON'
```

Comment:
There are many ways to set something to true in CMake, like `1`, `true`, `yes`, `TRUE`, `On`, `ON`, etc. This will not work unless one uses `ON` literally.

Reply:
So is it ok if I put `!= 'OFF'`?
According to line 141 of cmake/modules/RootBuildOptions.cmake it should be always literally `OFF` if experimental pyroot is not enabled.